index.html

```diff
@@ -247,6 +247,54 @@
         particular, the algorithms defined in this specification are intended
         to be easy to follow, and not intended to be performant.)
       </p>
+      <section>
+        <h3>
+          Conformance Classes
+        </h3>
+        <p>
+          This specification describes the conformance criteria for two classes
+          of user agents.
+        </p>
+        <dl>
+          <dt>
+            <dfn>Controlling user agent</dfn>
+          </dt>
+          <dd>
+            <p>
+              Web browsers that conform to the specifications of a
+              <a>controlling user agent</a> must be able to start and control
+              presentations by providing a <a>controlling browsing context</a>
+              as described in this specification. This context MUST implement
+              the <a><code>PresentationConnection</code></a>,
+              <a><code>PresentationConnectionAvailableEvent</code></a>, and
+              <a><code>PresentationRequest</code></a> interfaces.
+            </p>
```

Comment:
s/This context MUST implement/A controlling user agent implements/
In other words, it is the user agent that implements the interfaces and I would not include normative statements directly in the definition of the conformance class, but merely state facts.

The "Presentation" interface should be in that list, too.